src/js/api/set-config.js

```diff
@@ -19,30 +10,55 @@ function setAutoStart(model, controller, autoStart) {
 
 export default (controller, newConfig) => {
     const model = controller._model;
+    const attributes = model.attributes;
 
-    if (!size(newConfig)) {
-        return;
+    if (newConfig.height) {
```

Comment:
thoughts on maybe breaking this out into a diff func? feels a little strange to have 2 separate config mutators in the same func which seem responsible for different ways of attacking the problem (explicit logic around sizing vs a switch case depending on attribute)

Reply:
`width` `height` and `aspectratio` need to be handled at once. The only place where it makes sense to do this is in `view.resize` but I don't want to change it at this point. These cannot be handled one at a time in the switch statement - since we could end up resizing 3 times.

Reply:
totally agree, was more curious if you thought it helpful to break that out into another function for code clarity purposes